h2olog.h

```diff
@@ -27,21 +27,34 @@
 #include <vector>
 #include <bcc/BPF.h>
 
-typedef struct st_h2o_tracer_t {
+struct st_h2o_tracer_t;
+typedef struct st_h2o_tracer_t h2o_tracer_t;
+
+struct st_h2o_tracer_t {
     /*
      * Where to output the results. Defaults to `stdout`.
      */
-    std::FILE *out;
+    FILE *out;
+
+    /*
+     * The number of events emitted in `handle_event`.
+     */
+    size_t count;
 
     /*
-     * The number of events emitted  in `handle_event`.
+     * The number of lost events. It is reset periodically.
      */
-    std::size_t count;
+    size_t lost_count;
```

Comment:
Is `uint64_t` preferable than `size_t`. Because argument of lost callback(`perf_reader_lost_cb`) is passed as `uint64_t`

Reply:
It makes sense. We should use `uint64_t` here for portability.

bcbd96ade201b3b9232ebd092f85323a2ce8d260